lab-kaylyn/app/component/navbar/_navbar.scss

```diff
@@ -0,0 +1,42 @@
+form {
+  text-align: center;
+
+  .form-control {
+    height: 3em;
+  }
+}
+
+.navbar {
+  .btn-logout {
+    display: none;
+    @media (min-width: 500px){
+      display: inline-block;
+    }
+  }
+
+  .btn-dropdown  {
+    padding: 10%;
+    span{
+      font-size: 1.5em;
+
+      @media (min-width: 500px){
+        display: none;
+      }
+    }
+  }
+}
+
+
+
+@media (min-width: 768px) {
+  .form-inline .form-control {
+    display: inline-block;
+    width: 35em;
+  }
+}
+
+.home .gallery-list-container h2 {
+    margin-top: 3em;
```

Comment:
@kaylynyuh great use of `em` - try using `vw` and/or `vh` for slick font scaling too